Featutre/basic fastapi structure

IWOrLaFI opened the pull request.

IWOrLaFI committed:
create project

IWOrLaFI committed:
edit settings.py

Rybasher commented on requirements.txt:
```diff
@@ -0,0 +1,4 @@
+fastapi
+uvicorn
+pydantic
+python-dotenv
```
Создай виртуальное окружение, а для добавления зависимостей в requirements.txt используй команду
`pip freeze requirements.txt`

IWOrLaFI replied:
готово

IWOrLaFI committed:
edit requirements.txt

IWOrLaFI committed:
refactoring app.py to dir routes
 edit __main__.py
    'workshop.app:app' -> 'workshop.routes.app:app'

IWOrLaFI committed:
refactoring app.py to dir workshop
 edit __main__.py
     'workshop.routes.app:app'->'workshop.app:app'
     added test_routers.py

IWOrLaFI committed:
added config.py
